src/dotnet/Chat.Service/Controllers/MessageController.cs

```diff
@@ -13,108 +7,30 @@ public class MessageController : ControllerBase
 {
     private readonly ISessionResolver _sessionResolver;
     private readonly ICommander _commander;
-    private readonly ILogger<MessageController> _log;
 
-    public MessageController(ISessionResolver sessionResolver, ICommander commander, ILogger<MessageController> log)
+    public MessageController(ISessionResolver sessionResolver, ICommander commander)
     {
         _sessionResolver = sessionResolver;
         _commander = commander;
-        _log = log;
     }
 
     [HttpPost]
-    [DisableFormValueModelBinding]
     [Route("api/chats/{chatId}/message")]
-    public async Task<ActionResult<long>> PostMessage(ChatId chatId)
+    public async Task<ActionResult<long>> PostMessage(ChatId chatId, [FromBody] ChatMessage chatMessage)
```

Comment:
Let's keep old contract for some time and make new method that breaks contract otherwise to keep backward compatibility of mobile versions

Reply:
I think it's OK to skip this now. BUt in prod we have to do this